doc/source/whatsnew/v1.1.0.rst

```diff
@@ -733,6 +627,115 @@ apply and applymap on ``DataFrame`` evaluates first row/column only once
 
     df.apply(func, axis=1)
 
+.. _whatsnew_110.api.other:
+
+Other API changes
+^^^^^^^^^^^^^^^^^
+
+- :meth:`Series.describe` will now show distribution percentiles for ``datetime`` dtypes, statistics ``first`` and ``last``
+  will now be ``min`` and ``max`` to match with numeric dtypes in :meth:`DataFrame.describe` (:issue:`30164`)
+- Added :meth:`DataFrame.value_counts` (:issue:`5377`)
```

Comment:
why are you lumping enhancements in here? this is very confusing

Reply:
Well, this are the entries that were originally (look at the diff above) under "Other API changes".

I fully agree that this is an enhancement, and that should be cleaned-up, see https://github.com/pandas-dev/pandas/issues/34801, but here I am just correcting the order of the sections, not actually fixing the bullet points itself as mentioned in the OP)

Reply:
ok fair enough, yeah these should really be separated a more fine grained way

Reply:
Yeah, value_counts here should simply be moved to the enhancement section, and we can discuss in #34801 what kind of more fine grained sections we want for the API changes itself.